src/widgets/Card/Card.tsx

```diff
@@ -0,0 +1,37 @@
+import { NavLink, useSearchParams } from 'react-router-dom';
+import { Character, CharactersProps } from '../../types';
+import { getCharacterId } from '../../utils/getCharacterId';
+import './Card.css';
+
+const Cards = (props: CharactersProps): JSX.Element => {
```

Comment:
```suggestion
const Cards:React.FC<CharactersProps> = ({characters}) => {
```
use generics and destructuring